### PR TITLE
Add JSON endpoint for inline book component

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Needed for the upcoming native inline book.

picture 2

### CODE REVIEW

File: commercial/app/controllers/BookOffersController.scala [hunk 1]
```diff
@@ -78,6 +78,36 @@ class BookOffersController(bookFinder: BookFinder, bestsellersAgent: Bestsellers
     booksSample(specificIds, segment) map result
   }
 
+  def getBook = Action.async { implicit request =>
+    specificId map { isbn =>
+      bookFinder.findByIsbn(isbn) map {
+        _ map { book =>
+          val json = Json.toJson(book)
+          Cached(60.seconds){
```
Comment (kelvin-chappell) [inline commercial/app/controllers/BookOffersController.scala]: Is the 60s specifically for this route or should it be the same as the other routes?  ie share the value

**Review by kelvin-chappell — APPROVED** [review]
👍

File: commercial/app/controllers/BookOffersController.scala (hunk 1 above)
Comment (regiskuckaertz) [inline commercial/app/controllers/BookOffersController.scala]: This is the same for all endpoints I think. What is the best place to store that value? In the config?

Comment (kelvin-chappell) [inline commercial/app/controllers/BookOffersController.scala]: It looks like there's a value here: https://github.com/guardian/frontend/blob/master/commercial/app/controllers/package.scala#L12

**Comment by regiskuckaertz** [comment]
[Native Inline book JSON endpoint](https://trello.com/c/2t27A7M3/170-native-inline-book-json-endpoint)

**Pull request merged**